irohad/consensus/yac/timer.hpp

```diff
@@ -20,6 +20,8 @@
 
 #include <functional>
 
+//TODO: This timer interface should not be under YAC. It should be under something like utils and made available
```

Comment:
This interface provides the methods required only by YAC, so it may not be sufficient for other classes. Class which implements Timer may reuse Rx API.